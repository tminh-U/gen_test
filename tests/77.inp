10
7 1 11 14 6 13 12 17 5 9 
8
6 10
9 9
6 9
6 8
6 9
10 10
6 6
9 9
