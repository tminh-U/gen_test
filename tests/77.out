9
2
7
6
7
2
2
2
